composer.json

```diff
@@ -14,11 +14,11 @@
         }
     ],
     "require": {
-        "php": "^5.3.3 || ^7.0 || ^8.0",
+        "php": ">=7.2",
```

Comment:
Why not `^7.2 || ^8.0`?

Reply:
Using `>=7.2` means: You can use this library with 7.2 or any later version of PHP. If you notice something broken in PHP9, feel free to submit a PR. 

Using `^7.2 || ^8.0` means: I have only tested this in PHP 7.2 and PHP 8.*. When PHP 9 comes out I will be responsible to fix all the issues. 

-------

So, technically there is no difference at the moment, but `>=7.2` is more welcoming for those cool people that running bleeding edge PHP versions. It will also be easier for them to test out this library on PHP9.

Reply:
Im fine with either way, but I think the `>=7.2` is better.

Reply:
I agree that `>=` makes it easier to test on php 9. On the other hand, composer 2 has made it easier to just ignore the php requirement, and i think the people who are running bleeding edge understand how to do that. `^` is a bit more restrictive, but makes it harder for users to accidentally install a version that breaks a lot of things.

PHP  9 is (probably) multiple years away, but i'd prefer to declare compatibility with it when that time comes.

Reply:
Sure